core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -137,6 +136,14 @@ public static String start(String hostIpAddress, DockerClient client) {
                             }
                         } catch (IOException e) {
                             log.warn("Can not connect to Ryuk at {}:{}", hostIpAddress, ryukPort, e);
+
+                            try {
+                                // sleep for a moment to avoid excessive log spam
+                                Thread.sleep(500);
```

Comment:
we have RateLimiter available on classpath, could you please consider using it instead?
Also, 500ms is rather long initial delay.

Reply:
yes, of course.